.github/workflows/ci-schedule-compatibility.yaml

```diff
@@ -11,6 +11,7 @@ jobs:
     if: ${{ github.repository == 'karmada-io/karmada' }}
     runs-on: ubuntu-22.04
     strategy:
+      max-parallel: 2
```

Comment:
Is `2` concurrency actually only 2 instances running Karmada? I feel like the numbers are a bit conservative.

Reply:
Yes, only two, as @liangyuanpeng tested, we  can set it as 5 or 6